src/reconcile/rec_write.c

```diff
@@ -4905,16 +4862,12 @@ __rec_col_var(WT_SESSION_IMPL *session,
 		}
 
 		/*
-		 * If we had a reference to an overflow record we never used,
+		 * The first time we find an overflow record we never used,
 		 * discard the underlying blocks, they're no longer useful.
-		 *
-		 * One complication: we must cache a copy before discarding the
-		 * on-disk version if there's a transaction in the system that
-		 * might read the original value.
 		 */
 		if (ovfl_state == OVFL_UNUSED &&
 		    vpack->raw != WT_CELL_VALUE_OVFL_RM)
-			WT_ERR(__wt_ovfl_cache(session, page, upd, vpack));
+			WT_ERR(__wt_ovfl_remove(session, page, upd, vpack));
```

Comment:
Do the two code paths that call `__wt_ovfl_remove` from reconciliation guarantee that the reconciliation is an update-restore? Otherwise I'm missing something about this change.